.ci/containers/contributor-checker/check-contributor.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+if [ -z "$GITHUB_TOKEN" ]; then
+    echo "Did not provide GITHUB_TOKEN environment variable."
+    exit 1
+fi
+if [ $# -lt 1 ]; then
+    echo "Usage: $0 pr-number"
+    exit 1
+fi
+PR_NUMBER=$1
+
+USER=$(curl -H "Authorization: token ${GITHUB_TOKEN}" \
+  "https://api.github.com/repos/GoogleCloudPlatform/magic-modules/issues/${PR_NUMBER}" | jq .user.login)
+
+# This is where you add users who do not need to have an assignee chosen for
+# the.
+if $(echo $USER | fgrep -wq -e ndmckinley -e danawillow -e emilymye -e erjohnso -e geojaz -e megan07 -e paddcarver -e rambleraptor -e SirGitsalot -e slevenick -e c2thorn); then
```

Comment:
Do we know geojaz?

```suggestion
if $(echo $USER | fgrep -wq -e ndmckinley -e danawillow -e emilymye -e erjohnso -e geojaz -e megan07 -e paddycarver -e rambleraptor -e SirGitsalot -e slevenick -e c2thorn -e rileykarson); then
```

Reply:
I totally don't, but he's on the list of contributors to magic-modules.

Reply:
Eh, he's made one PR (https://github.com/GoogleCloudPlatform/magic-modules/pulls?q=author%3Ageojaz). I say we remove him, and add him back later if need be.

Reply:
Done!